### ISSUE TITLE
link preload in iOS 11.4 not working

### ISSUE THREAD

**Issue by GitToTheHub** [Fyrd/caniuse#5113]
When i add a preload link to my html like
`<link rel="preload" as="style" href="main.css" onload="alert('onload preload')"/>`
onload is never fired and i see no alert dialog. At caniuse the minimum iOS Version is 11.4. Can somebody confirm it's really working in 11.4? I tested in the simulator and it was not successful.

**Comment by kiding** [comment]
https://github.com/Fyrd/caniuse/blob/294ce7254e4f3e9f5e7395b4894ecf21d7eefc9d/CONTRIBUTING.md#L58
https://github.com/Fyrd/caniuse/blob/294ce7254e4f3e9f5e7395b4894ecf21d7eefc9d/CONTRIBUTING.md#L52

"Flagged(d)" typically means the feature needs to be enabled by the user, and "supported(y)" means if it is enabled by default. Link Preload has been ["on-by-default experimental feature" since iOS 11.3.](https://trac.webkit.org/browser/webkit/releases/Apple/iOS%2011.3/WebKit/Shared/WebPreferences.yaml)

There seems to be no way in CanIUse to differentiate if a feature is 1) on-by-default experimental 2) off-by-default experimental. I could think of some reasons: 1) and "official support" is essentially the same thing to most developers, and browsers could disagree over the term "experimental." Overall, surveying all the browsers for data only to differentiate 1) and 2) might be somewhat overkill in my opinion.

**Comment by GitToTheHub** [comment]
Hi, thank you for your good response. Thank you too for the link to https://trac.webkit.org/
I looked for the iOS version, when it's no longer experimental and it's starting with iOS 12. I'm developing in Apps with Cordova and so my Apps are using HTML in a WebView. It seems, it has no impact, if a experimental feature is enabled by default, when you using the UIWebView. Preloading worked in Safari but not in the App's Webview. I see you can add comments to caniuse. Maybe you could add the Info, that it's an experimental feature in iOS 11.3-11.4 enabled by default and is fully integrated since iOS 12.

**Comment by kiding** [comment]
Ah, that’s interesting. I wonder if the difference in UI/WKWebView applies to the other experimental features, or it’s some fluke around only this feature. I couldn’t find any related documentation or WWDC sessions, so some further testing might be needed.

**Comment by GitToTheHub** [comment]
Or maybe it's a problem with the obsoleted UIWebView. Maybe in WKWebView there is no problem. But i'm not able to test it currently :/ For UIWebView i think the problem applies to all experimental features. I cannot imagine it's a problem with only this feature.

**Comment by Fyrd** [comment]
Yeah, @kiding is correct. As long as all users of the browser have it enabled it's just "supported". May be worth adding a note that it's considered experimental if that matters in certain uses of the engine.

**Comment by GitToTheHub** [comment]
I tested if the experimental features have impact to the WKWebView and it doesn't change anything if I switch something on/off. For me this behaviour make sense, because a user could accidently changing the behaviour of an app, when he is changing something there, and an App which uses WKWebView and for example relies on link preloading would break.

**Comment by GitToTheHub** [comment]
Forget my last comment about the WKWebView which i deleted now.

When i was testing with WKWebView, my Environment was mistakenly not using the WKWebView and still used the UIWebView. Here are my test results with WKWebView now.

In WKWebView link preloading works in iOS 11.4 no matter if the user is switching it on or off in Safari > Experimental Features.

But in UIWebView it is not working, no matter if it is turned on or off.
